.circleci/config.yml

```diff
@@ -276,7 +280,7 @@ jobs:
       - run:
           name: Delete non-signed archive
           command: |
-            rm << parameters.artifact_dir >>/slack_*_macOS_64-bit.tar.gz
+            rm << parameters.artifact_dir >>/slack_*_macOS_*.tar.gz
```

Comment:
🪓 An initial removal of this step missed that this step is most useful when codesigning happens before generating archives!

We continue to remove unsigned archives to repackage tarballs with the signed zip:

![codesign](https://github.com/user-attachments/assets/9b5d52de-5459-4373-acc9-374e485b304f)